src/main/java/org/jetbrains/bsp/bazel/server/BepStreamObserver.java

```diff
@@ -0,0 +1,57 @@
+package org.jetbrains.bsp.bazel.server;
+
+import com.google.devtools.build.v1.PublishBuildToolEventStreamRequest;
+import com.google.devtools.build.v1.PublishBuildToolEventStreamResponse;
+import io.grpc.stub.StreamObserver;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class BepStreamObserver implements StreamObserver<PublishBuildToolEventStreamRequest> {
+
+  private static final Logger LOGGER = LogManager.getLogger(BepStreamObserver.class);
+  private static final String BUILD_EVENT_TYPE_URL =
```

Comment:
This can be moved to the constants file

Reply:
It's specific for this class, not entire project - in my opinion it should stay in this class